README.md

```diff
@@ -0,0 +1,6 @@
+# SpikesLib - Examples
+Examples of how to use the [SpikesLib](https://github.com/Spikes-2212-Programming-Guild/SpikesLib2) library.
+
+## Includes:
+- Several subsystems and a drivetrain each with its own command examples.
+- Each subsystem has a RobotMap class, which holds the ports for the robot's devices.
```

Comment:
add the fact that they can just copy-paste the code, and only change the `RobotMap` for it to work